analysers/analyser_merge_parking_FR_BNLS

```diff
@@ -0,0 +1,66 @@
+## This program is free software: you can redistribute it and/or modify  ##
+## it under the terms of the GNU General Public License as published by  ##
+## the Free Software Foundation, either version 3 of the License, or     ##
+## (at your option) any later version.                                   ##
+##                                                                       ##
+## This program is distributed in the hope that it will be useful,       ##
+## but WITHOUT ANY WARRANTY; without even the implied warranty of        ##
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         ##
+## GNU General Public License for more details.                          ##
+##                                                                       ##
+## You should have received a copy of the GNU General Public License     ##
+## along with this program.  If not, see <http://www.gnu.org/licenses/>. ##
+##                                                                       ##
+###########################################################################
+
+from .Analyser_Merge import Analyser_Merge, Source, CSV, Load, Mapping, Select, Generate
+
+
+class Analyser_Merge_Parking_FR_BNLS(Analyser_Merge):
+    def __init__(self, config, logger = None):
+        Analyser_Merge.__init__(self, config, logger)
+        self.missing_official = self.def_class(item = 8130, id = 1, level = 3, tags = ['merge', 'parking'],
+            title = T_f('{0} parking not integrated', 'BNLS'),
+            detail = T_(
+'''This parking is referenced in the database of car parks managed by local authorities in France, off-street.
+This database is made up of local authorities and is available on data.gouv.fr.'''),
+            fix = T_(
+'''See [the mapping](https://wiki.openstreetmap.org/wiki/France/data.gouv.fr/Base_nationale_des_lieux_de_stationnement)
+on the  wiki. Add a node or add tags if already existing.'''))
+
+        self.init(
+            u"https://www.data.gouv.fr/fr/datasets/base-nationale-des-lieux-de-stationnement/",
+            u"Base Nationale des Lieux de Stationnement",
+            CSV(Source(attribution = u"équipe du Point d'Accès National", millesime = "2020/04/30",
+                    fileUrl = u"https://static.data.gouv.fr/resources/base-nationale-des-lieux-de-stationnement/20200430-143152/20200430-bnls.csv", encoding = "charset=unknown-8bit")),
+            Load("Xlong", "Ylat",
+                xFunction = self.float_comma,
+                yFunction = self.float_comma),
+            Mapping(
+                select = Select(
+                    types = ["nodes", "ways"],
+                    tags = {"amenity": "parking"}),
+                conflationDistance = 200,
+                generate = Generate(
+                    static1 = {
+                        "amenity": "parking"},
+                    static2 = {
+                        "source": self.source},
+                    mapping1 = {
```

Comment:
mapping1 = {
                         "ref:FR:BNLS": "id",

Reply:
i didn't make a "suggestion" as you learn me, because i have some difficulties to align (indentation ?). But i suppose web editor it's not the best editor !!
It's better to download all files and upload after ?

Reply:
You can re-edit your sugg until is right

Reply:
```suggestion
                    mapping1 = {
                        "ref:FR:BNLS": "id",
```